tests/rpc_test.go

```diff
@@ -754,11 +754,11 @@ func TestEth_EstimateGas(t *testing.T) {
 	require.NotNil(t, rpcRes)
 	require.NotEmpty(t, rpcRes.Result)
 
-	var gas hexutil.Bytes
+	var gas string
 	err := json.Unmarshal(rpcRes.Result, &gas)
 	require.NoError(t, err, string(rpcRes.Result))
 
-	require.Equal(t, "0xffdf", gas.String())
+	require.Equal(t, "0x1051d", gas)
```

Comment:
not sure if we should address this? technically it should remain the same?

Reply:
yeah it should likely be the same, do we have any tests for actual gas consumption vs estimation? if not we should add some